src/script/view_model/content/MessageListViewModel.ts

```diff
@@ -84,7 +84,6 @@ export class MessageListViewModel {
     private readonly userState = container.resolve(UserState),
     private readonly conversationState = container.resolve(ConversationState),
   ) {
-    this.mainViewModel = mainViewModel;
```

Comment:
not needed?

Reply:
No, as we already say `private readonly mainViewModel` in the constructor params.